presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -21,6 +21,7 @@ databases:
     warehouse_directory_path: /user/hive/warehouse
     inject_stats_for_immutable_tables: true
     inject_stats_for_mutable_tables: true
+    enforce_non_transactional_tables: false
```

Comment:
This should possibly be consumed by table fulfillers which are part of Tempto itself. Not only by the code here. But it can be a follwoup.